packages/ember-htmlbars/lib/keywords/get.js

```diff
@@ -35,8 +35,10 @@ let DynamicKeyStream = BasicStream.extend({
 
   key() {
     let key = this.keyDep.getValue();
-    if (typeof key === 'string') {
+    if (typeof key === 'symbol') {
```

Comment:
Can do! Is there an example somewhere of how to check for browser support within tests? Thanks